pkg/blobplugin/Dockerfile

```diff
@@ -14,24 +14,29 @@
 
 FROM registry.k8s.io/build-image/debian-base:bullseye-v1.4.3
 
-ARG ARCH=amd64
-ARG binary=./_output/${ARCH}/blobplugin
+ARG TARGETARCH
+ARG binary=./_output/${TARGETARCH}/blobplugin
 COPY ${binary} /blobplugin
 
 RUN mkdir /blobfuse-proxy/
 
 COPY ./pkg/blobfuse-proxy/init.sh /blobfuse-proxy/
 COPY ./pkg/blobfuse-proxy/blobfuse-proxy.service /blobfuse-proxy/
-COPY ./_output/${ARCH}/blobfuse-proxy /blobfuse-proxy/
+COPY ./_output/${TARGETARCH}/blobfuse-proxy /blobfuse-proxy/
 
 RUN chmod +x /blobfuse-proxy/init.sh && \
  chmod +x /blobfuse-proxy/blobfuse-proxy.service && \
  chmod +x /blobfuse-proxy/blobfuse-proxy
 
-RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates uuid-dev util-linux mount udev wget e2fsprogs nfs-common netbase
+RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates uuid-dev util-linux mount udev wget e2fsprogs nfs-common netbase procps conntrack iptables bind9-host iproute2 bash netcat sysvinit-utils
```

Comment:
add a comments that `procps conntrack iptables bind9-host iproute2 bash netcat sysvinit-utils` are only needed by aznfs package.